nixos/modules/programs/noisetorch.nix

```diff
@@ -0,0 +1,25 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let cfg = config.programs.noisetorch;
+in {
+  options = {
+    programs.iotop.enable = mkEnableOption "noisetorch + setcap wrapper";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.noisetorch;
+      description = ''
+        The package to use.
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    security.wrappers.noisetorch = {
+      source = "${pkgs.noisetorch}/bin/noisetorch";
+      capabilities = "cap_sys_resource=+ep";
+    };
+  };
```

Comment:
This doesn't adds the desktop files necessary for it to show in menu (`noisetorch`  is a GUI application), so it can only be started by terminal. It would be interesting to have a way to add this without using `environment.systemPackages` (since this would also add the non wrapped command to the PATH and cause conflicts).